tests/cases/care_partner_platform/case_care_platform_events.yaml

```diff
@@ -303,6 +303,128 @@ TestCarePartnerCreateEventForParticipant:
   # The participant’s device schedule will now also reflect this new event with all the details as entered.
+    - Type: wait_for
+      Role: localiOS
+      Strategy: class_chain
+      Id: $PAGE.seniors_app_home.event_based_on_name_element$
+
+# CXTB-132: Call Portal View - Edit Event for Participant.
+TestCarePartnerEditEventForParticipant:
+  Environment: Settings
+  Precases:
+    - LoginNeverAloneHelper
+    - SeniorCleanPrompts
+  Aftercases:
+    - TerminateApp
+  Roles:
+    - Role: desktopChrome
+      App: desktop
+    - Role: localiOS
+      App: NeverAlone
+  Actions:
+  # Create an event for today in the future.
+    - Type: case
+      Value: SetScheduledCallForSeniorsToStartIn11Minutes
+  # Calling Login case.
+    - Type: case
+      Value: CarePartnerLogin
+  # Navigate to Participants.
+    - Type: case
+      Value: CarePartnerNavigateToParticipants
+  # Click on the corresponding participant.
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant.senior_complete_name$
+  # Click on the "Call Portal" button.
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_participant.call_portal_btn$
+  # Page updates to show portal screen with "No active call" text in video window.
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.video_status$
+    - Type: get_attribute
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.video_status$
+      Greps:
+        - var: CALL_MESSAGE
+          attr: textContent
+          condition: nempty
+          match: "(.*)"
+    - Type: assert
+      Asserts:
+        - Type: eq
+          Var: CALL_MESSAGE
+          Value: 'No active call.'
+  # Calendar shows today's date by default, Select the date of the event created for this test.
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.participant_schedule_based_on_element$
+  # Expand the event.
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.participant_schedule_based_on_element$
+  # Verify that the row should now show the Edit and Delete buttons.
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.edit_event_button$
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.delete_event_button$
+  # Click the "Edit" button.
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.edit_event_button$
+  # "Edit event" window appears to the left of the Events panel, showing all information as currently entered.
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.care_platform_call_portal.edit_event_modal_header$
+  # You should not be able to Save and close the event unless all of these fields have been populate:
+  # Subject, Date, Start Time and End Time
+    - Type: loop
+      Times: 50
+      Case: CarePartnerClearSubjectInputField
```

Comment:
Could you please try with this:

    - Type: clear_field
      Strategy: xpath
      Id: $PAGE.care_platform_call_portal.subject_input$
      
Instead of the looped case.

Reply:
I tried but was not successful, the action is properly built but the Care Platform input field did not clear after all. I did a solution similar to one on this file introduced on this commit: https://github.com/neveralone-chs-org/TestRay/blob/5d26c62e8f69f430972eb3e7cb51c2f24f987b26/tests/cases/case_employee_health_portal_account.yaml on line 204.